src/index.js

```diff
@@ -426,7 +410,25 @@ for (let i = 0; i < (NUM_TRIALS + 1); i++) {
     },
   };
 
-  practice_trials.push(practice_block, feedback_trial);
+  const practice_block_visual_feedback = {
+    type: jsPsychHtmlKeyboardResponse,
+    is_html: true,
+    stimulus: () => {
+      const response = jsPsych.data.get().last(1).values()[0].response;
```

Comment:
Could you please add a comment here explaining what this line does? Something like:

```
// fetch the value of the keyboard response from the previous block
```